pkg/cases/handler.go

```diff
@@ -37,9 +54,14 @@ func (h *CaseHandler) GetAllCases(c echo.Context) error {
 // @Success 200 {object} api.ReturnedData{data=Case}
 // @Router /patient/{patientID}/case [GET]
 func (h *CaseHandler) GetLastCaseByPatientID(c echo.Context) error {
-	// ...
+	db := utils.GetDB()
+	db.Where("PatientID = ?", c.Param("patientID"))
```

Comment:
Is that an elegant design to put GORM operations in the router handler?

Reply:
I think it's ok to do so? In fact there's nothing to do in the handler if we put db operations elsewhere. The auth part should be done via middleware and the validation could be somewhere wraped in the `json.decoder`.

Reply:
> I think it's ok to do so? In fact there's nothing to do in the handler if we put db operations elsewhere. The auth part should be done via middleware and the validation could be somewhere wraped in the `json.decoder`.

Are `middleware` and `json.decoder` sort of plugins to Echo?

Reply:
by the way, is there any plan to implement the jwt validation recently.   It's a quite common part in all 4 groups' designs

Reply:
Middleware is a sort of design pattern, and is provided by echo. `json.decoder` is simply the process of extracting data from requests. For now you could simply ignore the jwt validation, or ask for an API for that. The detailed implementation does not affect us.